src/types.ts

```diff
@@ -709,6 +709,10 @@ export interface PushDelete {
   deviceId: string,
 }
 
+export interface AddressAtIndexRequest {
+  index: number,
```

Comment:
This parameter is optional, right?
```suggestion
  index?: number,
```

Reply:
It is, thanks!

Fixed in 8b3a064baafd4fcd5dedc94b242b4613aa8c7792